labs/sequential-events-targeting/README.md

```diff
@@ -0,0 +1,199 @@
+# Sequential Events Audience Targeting
+
+A typical use case for behavioural targeting would be targeting users based on a
+sequence of events in a specific order.
+
+An example would be targeting users who have added items to their cart and
+haven't yet checked out, thus creating an "abandoned cart" campaign.
+
+Having this functionality also enables us to create personalisation campaigns
+that run _until a user has completed an action_. For example, running a
+promotion for 10% off your next order. Once each user has seen the campaign, and
+completed an order they should be removed from the campaign.
+
+## Pre-requisites
+
+Optimizely Personalisation and behavioural targeting is part of Optimizely Web
+Enterprise plans.
+
+## Instructions
+
+### Step 1: Create a custom event called `exposed_to_campaign`
+
+Create a custom event by navigating to your Optimizely project, selecting
+"Implementation" in the menu, navigating to the "Events" tab and click "Create
+New Event...".
+
+Configure your event as below:
+
+![Creating a custom event in Optimizely
+Web](https://raw.githubusercontent.com/optimizely/labs/master/labs/sequential-events-targeting/custom-event.png)
+
+We are going to use this event each time a user is bucketed into an experiment
+or personalisation campaign. When we create a custom event we get the ability to
+query those events using the Optimizely Behavioural API. This gives us the
+timestamp for the event, which we can use to compare the chronological sequence
+of events.
+
+The Behavioural API also gives us functions for sorting, reducing and querying
```

Comment:
nit: Maybe lowercase Behavioural (or be consistent throghtout).

Also, might be worth linking to the docs the first time we mention this API or perhaps link in the prerequisite section.